### DISCUSSION TITLE
Pluto Language Server

### DISCUSSION THREAD

**Discussion by Apis035** [PlutoLang/Pluto#756] [Q&A] [answered]
It seems that [Pluto Language Server](https://github.com/PlutoLang/Language-Server) is archived and the extension does not exist on VSCode Marketplace (only syntax highlighting exists). What is the current status of this tool?

**Comment by Sainan** [comment, marked as answer]
The language server was an experiment. Ultimately, we found it to produce more problems than it solved, which is not in small part due to the very architecture of LSP.

The syntax highlighting package has also matured quite a bit, especially considering that it now has build systems, which allows for easy code execution and validation by simply pressing Ctrl+B.

All in all, it seemed right to focus on the syntax highlighting package only, because it's solving problems quite effectively, and TextMate bundles are already proven to work.
